test/web-platform-tests/index.js

```diff
@@ -204,6 +204,9 @@ describe("Web Platform Tests", () => {
     "dom/events/Event-type-empty.html",
     "dom/events/EventListener-handleEvent.html",
     "dom/events/EventTarget-addEventListener.html",
+/*    "dom/events/EventListenerOptions-capture.html",*/
+    "dom/events/AddEventListenerOptions-once.html",
+/*    "dom/events/AddEventListenerOptions-passive.html",*/
```

Comment:
We should either support passive and pass the tests, or not have any code for it.